src/samples/test-bots/OBOAuthorization/README.md

```diff
@@ -13,19 +12,49 @@ This Agent has been created using [Microsoft 365 Agents Framework](https://githu
 
 ## Running this sample
 
-1. [Create an Azure Bot](https://aka.ms/AgentsSDK-CreateBot)
-   - Record the Application ID, the Tenant ID, and the Client Secret for use below
-
-1. **This needs documented to account for exchangeable tokens**  
-   1. Create App Registration
-      1. authorization
-      1. API Permissions
-      1. Expose API
-   1. Create Azure Bot **OAuth Connection**
-
-1. Configuring the token connection in the Agent settings
+1. Create a Agent in [Copilot Studio](https://copilotstudio.microsoft.com)
+   1. Publish your newly created Copilot
+   1. Go to Settings => Advanced => Metadata and copy the following values. You will need them later:
+      1. Schema name
+      1. Environment Id
+       
+2. [Create an Azure Bot](https://aka.ms/AgentsSDK-CreateBot)
+   - Record the Application ID, the Tenant ID, and the Client Secret for use in the "ServiceConnection" settings below.
+
+3. Setting up OAuth for an exchangeable token 
+   1. Create a new App Registration
+      1. SingleTenant
+      1. Give it a name and click **Register**
+      1. **Authentication** tab
+         1. **Add Platform**, then **Web**, Set `Redirect URI` to `Web` and `https://token.botframework.com/.auth/web/redirect`
+         1. **Add Platform**, then **Mobile and desktop applications**, and add an additional `http:localhost` Uri.
+      1. **API Permissions** tab
+         1. **Dynamics CRM** with **user_impersonation**
+         1. **Graph** with **User.Read**
+         1. **Power Platform API** with **CopilotStudio.Copilots.Invoke**
+         1. Grant Admin Consent for your tenant.
+      1. **Expose an API** tab
+         1. Click **Add a Scope**
+         1. **Application ID URI** should be: app://botid-{{appid}}
+         1. **Scope Name** is "defaultScope"
+         1. **Who can consent** is **Admins and users**
+         1. Enter values for the required Consent fields
+      1. **Certificates & secrets**
+         1. Create a new secret and record the value.  This will be used later.
+         
+4. Create Azure Bot **OAuth Connection**
+   1. On the Azure Bot created in Step #2, Click **Configuration** tab then the **Add OAuth Connection Settings** button.
+   1. Enter a **Name**.  This will be used later.
+   1. For **Service Provider** select **Azure Active Directory v2**
+   1. **Client id** and **Client Secret** are the values created in step #3.
+   1. Enter the **Tenant ID**
+   1. **Scopes** is "app://{{appid}}/defaultScope"
```

Comment:
this app:// or app://bot-id ?

Reply:
Hmmm. I'm betting it's whatever the used for "Application ID URI" in the previous step.  And we said to use "botid-".  Good catch.